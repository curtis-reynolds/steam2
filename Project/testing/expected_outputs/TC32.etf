03_validgamename_______sellstd_______100.00
00_sellstd_________SS_000000.00
