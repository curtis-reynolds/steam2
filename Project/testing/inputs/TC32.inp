login
sellstd
sell
validgamename
100
