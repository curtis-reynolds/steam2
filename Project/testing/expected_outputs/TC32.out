Login successful.
Game listed for sale.
